hal/lib/athena/FRCDriverStation.cpp

```diff
@@ -143,7 +143,7 @@ int32_t HAL_GetJoystickDescriptor(int32_t joystickNum,
       HAL_kMaxJoystickAxes; /* set to the desc->axisTypes's capacity */
   desc->buttonCount = 0;
   desc->povCount = 0;
-  int retval = FRC_NetworkCommunication_getJoystickDesc(
+  int32_t retval = FRC_NetworkCommunication_getJoystickDesc(
```

Comment:
These ones should be fine, as they're just status variables anyway. This should never be larger then 32 bits anyway.